src/controllers/category.js

```
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose')
+
+const categoryService = require('~/services/category')
+
+const { createBadRequestError, createNotFoundError } = require('~/utils/errorsHelper')
+
+const createCategory = async (req, res) => {
+  const categoryData = req.body
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)